Makefile.devcontainer

```diff
@@ -88,3 +88,6 @@ travis: test-race test-cover test-crosscover
 		PMM_TEST_RUN_UPDATE=1 \
 		make test-race-crosscover
 	make check
+
+ci-reviewdog:
+	golangci-lint run -c=.golangci.yml --out-format=line-number | reviewdog -f=golangci-lint -level=error -reporter=github-pr-check
```

Comment:
I think `error` will produce that many false positives that we will learn to ignore output completely. The trick we use in SaaS with two files with different sets of linters and levels is useful.

Reply:
Also there is the second type of reporter `github-pr-review`, it adds comments to code.

on SAAS we use `github-pr-check` for critical linters to broke CI pipeline and `github-pr-review` for non-critical.

Reply:
fixed